config/cloudinary.yml

```diff
@@ -0,0 +1,19 @@
+---
+development:
+  cloud_name: hckxckfb8
+  api_key: '482316417255674'
+  api_secret: 6LxzgOeveM3kd2_-oOksxVukYYc
+  enhance_image_tag: true
+  static_image_support: false
+production:
+  cloud_name: hckxckfb8
```

Comment:
@katyjane8 you might want to take this out again...

Reply:
should be fixed now - I need to keep adding to the .gitignore